packages/terra-alert/src/terra-dev-site/doc/example/LongTextExample.jsx

```diff
@@ -30,7 +32,7 @@ class LongTextExample extends React.Component {
         <span>
           Four score and seven years ago our fathers brought forth on this continent, a new nation, conceived in Liberty, and dedicated to the proposition that all men are created equal.
           <Toggle isOpen={!this.state.allTextShown}>
-            <Button onClick={this.handleShowMore} text="Show More" style={buttonStyle} />
+            <Button onClick={this.handleShowMore} text="Show More" className={cx('button')} />
```

Comment:
```suggestion
            <Button onClick={this.handleShowMore} text="Show More" />
```

This inline style was old and not correct - lets just remove the extra classname and have no additional styles applied (and remove the extra `LongTextExample.scss` css file). We'll fix later with an update using new `terra-show-hide` component.